resources/sass/pages/_torrents.scss

```diff
@@ -304,15 +312,15 @@
   .torrent-search--list__row {
     grid-template-areas:
       "poster format format overview overview overview overview overview overview"
-      "age age size size rating seeders leechers completed buttons";
+      "age age action action size size rating seeders leechers completed buttons";
```

Comment:
2 more columns were added, but `grid-template-columns` wasn't updated. Make sure to update `grid-template-columns` and verify all resolutions work with the updated dimensions.